src/Devtools.tsx

```diff
@@ -221,7 +240,17 @@ export const QueryStatus: Component<QueryStatusProps> = (props) => {
   const styles = getStyles();
 
   return (
-    <span class={styles.queryStatusTag}>
+    <span
+      onClick={() =>
+        setSelectedStatus((prev) =>
+          prev === props.label.toLowerCase() ? null : (props.label.toLowerCase() as any),
+        )
+      }
+      classList={{
+        [styles.queryStatusTag]: true,
+        [styles.selectedQueryStatusTag]: selectedStatus() === props.label.toLowerCase(),
+      }}
+    >
```

Comment:
Let's use cx everywhere instead of classList. I say this because we'll do this for react too and they dont have a classList property :D

Reply:
https://github.com/ardeora/solid-query-devtools/pull/2/commits/90f9e393b426ab1af22621f211c0031e3f76370c